x/dclauth/keeper/account.go

```diff
@@ -82,7 +113,7 @@ func (k Keeper) IterateAccounts(ctx sdk.Context, cb func(account types.Account)
 
 // Check if account has assigned role.
 func (k Keeper) HasRole(ctx sdk.Context, addr sdk.AccAddress, roleToCheck types.AccountRole) bool {
-	account, found := k.GetAccount(ctx, addr)
+	account, found := k.GetAccountO(ctx, addr)
```

Comment:
Why can't we use `GetAccount` instead so that returned `nil` is the same as `found=false`?